services/core-web/src/reducers/rootReducer.ts

```diff
@@ -0,0 +1,83 @@
+import { combineReducers } from "redux";
+import { createReducer } from "@common/utils/helpers";
+import mineAlertReducer from "./mineAlertReducer";
+import networkReducer from "./networkReducer";
+import { sharedReducer } from "@mds/common/redux/reducers/rootReducerShared";
+import * as reducerTypes from "@mds/common/constants/reducerTypes";
+import documentReducer from "./documentReducer";
+
+const coreReducer = {
```

Comment:
Reducers that only existed in Core combined with the shared ones from the common package